creator-node/Dockerfile

```diff
@@ -47,6 +48,8 @@ COPY . .
 COPY scripts/wait /usr/bin/wait
 RUN chmod +x /usr/bin/wait
 
+RUN ./node_modules/.bin/tsc --project ./
```

Comment:
is it worth documenting the diff components of this file a bit

Reply:
added some docs